app/models/setting/remote_execution.rb

```diff
@@ -67,7 +67,12 @@ def self.load_defaults
         self.set('remote_execution_cleanup_working_dirs',
                  N_('When enabled, working directories will be removed after task completion. You may override this per host by setting a parameter called remote_execution_cleanup_working_dirs.'),
                  true,
-                 N_('Cleanup working directories'))
+                 N_('Cleanup working directories')),
+        self.set('remote_execution_cockpit_href',
+                 N_('Where to find the Cockpit instance for the Web Console button.  By default, no button is shown.'),
```

Comment:
We might want to make it clear here that a relative URL is just fine, like the "/webcon" one I have been using with the reverse proxy.